### ISSUE TITLE
can´t sync

### ISSUE THREAD

**Issue by SandroIeva** [AkashaProject/Alpha#49]
try to sync the APP, but nothing happens. Iam using a macbook pro Late2012 and a good connection.

**Comment by SeverS** [comment]
@SandroIeva can you check if those 2 icons from the top have green borders?

Thanks

**Comment by SandroIeva** [comment]
the border are red

**Comment by SeverS** [comment]
Hello @Sandroleva,
Sorry for the delay.  Did you managed to make it work? If you need any help please let us know. 
Thanks

**Comment by BubbaBubbus** [comment]
Hello SeverS, 
i have the same problem. I am using Win10. 
Geth and IPFS service are off. How can i aktivate this service?

**Comment by provatidis** [comment]
Hello,
I cant sync either on two different Win10 pcs. Geth and IPFS icons are red, there are no settings and no logs.
I uninstalled, deleted everything and reinstalled multiple times trying both exe and zip versions.
I checked firewall and even netstat to find a blocked akasha connection but I can't find any.
I searched for *ipfs* on my hard drive and there is no folder or file by containing that string (Is it supposed to download binaries?).
There is no mist or parity currently running.

Anyone has any idea on what else to try? What ports are used? maybe another app is blocking it

edit: akasha 0.5.2 solved it. Something to do with non en-us locale
